packages/cli/src/commands/sandbox/sandbox-secret/sandbox_secret_set_command.test.ts

```diff
@@ -47,6 +49,8 @@ void describe('sandbox secret set command', () => {
 
   beforeEach(async () => {
     secretSetMock.mock.resetCalls();
+    // Fake a TTY in tests
+    process.stdin.isTTY = true;
```

Comment:
We should mock this in the ctors in the tests rather than mutate the global process.stdin as it could have side effects for other tests

Reply:
Good catch. Updated the test.